### PR TITLE
Updating workflows/microbiome/taxonomy-profiling-and-visualization-with-krona from 0.1 to 0.2

### PR DESCRIPTION
**Pull request by gxydevbot**
Hello! This is an automated update of the following workflow: **workflows/microbiome/taxonomy-profiling-and-visualization-with-krona**. I created this PR because I think one or more of the component tools are out of date, i.e. there is a newer version available on the ToolShed.

By comparing with the latest versions available on the ToolShed, it seems the following tools are outdated:
* `toolshed.g2.bx.psu.edu/repos/iuc/kraken2/kraken2/2.1.1+galaxy1` should be updated to `toolshed.g2.bx.psu.edu/repos/iuc/kraken2/kraken2/2.1.3+galaxy1`

The workflow release number has been updated from 0.1 to 0.2.

### CODE REVIEW

File: workflows/microbiome/taxonomy-profiling-and-visualization-with-krona/Taxonomy-Profiling-and-Visualization-with-Krona.ga [hunk 1]
```diff
@@ -53,7 +53,7 @@
         {
             "class": "Person",
             "identifier": "0000-0001-9852-1987",
-            "name": "Bérénice Batut",
+            "name": "B\u00e9r\u00e9nice Batut",
```
[the remark this answers is not in the document]
Comment (mvdbeek) [reply workflows/microbiome/taxonomy-profiling-and-visualization-with-krona/Taxonomy-Profiling-and-Visualization-with-Krona.ga:56]: Need to fix that on the Galaxy side

[the remark this answers is not in the document]
Comment (mvdbeek) [reply workflows/microbiome/taxonomy-profiling-and-visualization-with-krona/Taxonomy-Profiling-and-Visualization-with-Krona.ga:56]: yes

Comment (bebatut) [inline workflows/microbiome/taxonomy-profiling-and-visualization-with-krona/Taxonomy-Profiling-and-Visualization-with-Krona.ga:56]:
```suggestion
            "name": "Bérénice Batut",
```

File: workflows/microbiome/taxonomy-profiling-and-visualization-with-krona/CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## [0.2] 2024-06-28
```
Comment (bebatut) [inline workflows/microbiome/taxonomy-profiling-and-visualization-with-krona/CHANGELOG.md:3]:
```suggestion
## [0.1.1] 2024-06-28
```

File: workflows/microbiome/taxonomy-profiling-and-visualization-with-krona/Taxonomy-Profiling-and-Visualization-with-Krona.ga [hunk 3]
```diff
@@ -64,7 +64,7 @@
     ],
     "format-version": "0.1",
     "license": "MIT",
-    "release": "0.1",
+    "release": "0.2",
```
Comment (bebatut) [inline workflows/microbiome/taxonomy-profiling-and-visualization-with-krona/Taxonomy-Profiling-and-Visualization-with-Krona.ga:67]:
```suggestion
    "release": "0.1.1",
```

**Pull request closed without merging**